server/routes/api/place.py

```diff
@@ -763,3 +766,134 @@ def placeid2dcid():
                 result[inId] = dcid
         return Response(json.dumps(result), 200, mimetype='application/json')
     abort(404, 'no valid dcids found for %s' % place_ids)
+
+
+@cache.cached(timeout=3600 * 24, query_string=True)  # Cache for one day.
+@bp.route('/ranking_chart/<path:dcid>')
+def api_ranking_chart(dcid):
+    """
+    Gets the ranking data (including ranks, placename, value, date, and source) for a given place.
+
+    Return value example:
+    {
+        "statvar":{
+            "date":"2022-03",
+            "data":[
+                {
+                    "rank":1,
+                    "value":0.1,
+                    "placeDcid":"geoId/06",
+                    "placename":"California"
+                }
+            ],
+            "numDataPoints":1,
+            "exploreUrl":"/ranking/UnemploymentRate_Person_Rural/State/country/USA?h=geoId/06&unit=%",
+            "sources":[
+                "https://www.bls.gov/lau/"
+            ]
+        }
+    }
+    """
+    result = {}
+    # Get the parent place.
+    if not dcid == EARTH_DCID:
```

Comment:
nit: can we switch the order of the if statement and just do `if dcid == EARTH_DCID: ... else: ...`